src/README.md

```diff
@@ -96,6 +96,31 @@ npm run watch
 ```
 
 
+## Linting files
+
+When you push changes to GitHub they will be linted using the GitHub Super-Linter.
+
+It is possible to run the Super-Linter locally if you have Docker installed.
+
+First up pull the Super-Linter Docker image (this takes some time to download but only need to do this once or when you want to upgrade the version of the super-linter):
+
+```
+docker pull github/super-linter:latest
+```
+
+Then to run the linting do this:
+
+```
+npm run lint
```

Comment:
Is is a fair assumption that a contributor will have NPM installed? If not, then we might want to surface the Docker command here which this is a wrapper of.

Reply:
I think this is a fair prerequisite for most developers. For those working in a pure Docker environment (like I know you like to do) with no local install they can always get the full command from `package.json` and set up an alias. I think I'd rather keep the README clean.

Maybe I could add a piece under the Docker section but leave this as is?

Reply:
Yes, I am okay with this.

Reply:
Done!